src/app/demo/bootstrap/bootstrap.component.html

```diff
@@ -227,6 +227,55 @@ <h1 i18n>What is Angular?</h1>
     <div class="info">For most of the simple apps, you can just copy/paste the code above "as is"</div>
   </slides-slide>
 
+  <slides-slide id="bootstrapping" class="bootstrapping-more-info" milestone="Bootstrap3">
+    <h1>Bootstrapping: More Info (BONUS)</h1>
+    <h2>
+      How does bootstrapping work in Angular 2?
+    </h2>
+    <br>
+    <h2>1. Kicks off execution environment. <b>platformBrowserDynamic()</b> tells Angular that we are operating in the browser.</h2>
+    <slides-code-editor path="app.module.ts" [fontSize]="25"
+                        [code]="code.moduleBootstrapping.code.mainTs"
+                        [focus-highlight-match]="code.moduleBootstrapping.matches.bootstrap"></slides-code-editor>
+    <a href="https://angular.io/docs/ts/latest/guide/appmodule.html"
+       target="_blank">
+      Read more about root module and bootstrapping in Angular 2
+    </a>
+  </slides-slide>
+
+  <slides-slide id="bootstrapping" class="bootstrapping-more-info" milestone="Bootstrap3">
+    <h1>Bootstrapping: More Info (BONUS)</h1>
+    <h2>
+      How does bootstrapping work in Angular 2?
```

Comment:
There is a lot of `Angular 2` on this page. We're on Angular 4. Shouldn't we be referring to it as just `Angular`?